app/cli.h

```diff
@@ -123,7 +123,9 @@ void native(Ptr<Options> options) {
   std_input << std::cin.rdbuf();
   std::string input = std_input.str();
 
+
   ResponseOptions responseOptions;
+  responseOptions.qualityScores=true;
```

Comment:
space around `=`. How did this pass style-checks?

Reply:
This was fixed in ee8cca7 but was changed to check quality option, ef5f56b